backend/src/test/java/com/example/parkingspot/controller/CarControllerMVCTest.java

```diff
@@ -0,0 +1,139 @@
+package com.example.parkingspot.controller;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import com.example.parkingspot.entity.Car;
+import com.example.parkingspot.entity.Event;
+import com.example.parkingspot.entity.Person;
+import com.example.parkingspot.repository.CarRepository;
+import com.example.parkingspot.service.CarService;
+import com.example.parkingspot.service.EventService;
+import com.example.parkingspot.service.PersonService;
+
+import java.util.List;
+import java.util.Optional;
+
+@WebMvcTest(CarController.class)
+@AutoConfigureMockMvc(addFilters = false)
+public class CarControllerMVCTest {
+    
+    @MockBean
+    EventService eventService;
+
+    @MockBean
+    CarService carService;
+
+    @MockBean
+    CarRepository carRepository;
```

Comment:
Nothing breaking nor major that needs fixing. Cant se carRepository is used in CarController or any of the tests

Reply:
Fixed